lib/mvi/messages/add_person_message.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'socket'
+
+module MVI
+  module Messages
+    # Builds an MVI SOAP XML message for adding a user.
+    #
+    # Call with a user object and use `.to_xml` method to create the XML message
+    #
+    # Example:
+    #  message = MVI::Messages::AddPersonMessage.new(user).to_xml
+    #
+    class AddPersonMessage
+      SCHEMA_FILE_NAME = 'mvi_add_person_template.xml'
+
+      def initialize(user)
+        @user = user
+      end
+
+      def to_xml
+        template = Liquid::Template.parse(
+          File.read(File.join('config', 'mvi_schema', SCHEMA_FILE_NAME))
+        )
+
+        template.render!(build_content(@user))
+      end
+
+      private
+
+      def build_content(user)
+        current_time = Time.now.utc
```

Comment:
`Time.current` should return the time in UTC or w/e Rails is set to, rather than the server time.